source/backend/dal/Repositories/AcquisitionFileRepository.cs

```diff
@@ -52,14 +54,98 @@ public Paged<PimsAcquisitionFile> GetPage(AcquisitionFilter filter, HashSet<shor
-            var items = query
-                .Skip(skip)
-                .Take(filter.Quantity)
-                .ToArray();
+            var pageItems = queryResult.Skip(skip).Take(filter.Quantity).ToList();
+
+            return new Paged<PimsAcquisitionFile>(pageItems, filter.Page, filter.Quantity, queryResult.Count);
+        }
+
+        /// <summary>
+        /// Get an Acquisition Files Export based on Filter.
+        /// </summary>
+        /// <param name="filter"></param>
+        /// <param name="regions"></param>
+        /// <param name="filterPersonId"></param>
+        /// <returns>List of AcquisitionFileExportDto.</returns>
+        public List<AcquisitionFileExportDto> GetAcquisitionFileExport(AcquisitionFilter filter, HashSet<short> regions, long? filterPersonId = null)
+        {
+            // RECOMMENDED - use a log scope to group all potential SQL statements generated by EF for this method call
+            using var scope = Logger.QueryScope();
+
+            filter.ThrowIfNull(nameof(filter));
+            if (!filter.IsValid())
+            {
+                throw new ArgumentException("Argument must have a valid filter", nameof(filter));
+            }
+
+            var predicate = GenerateAcquisitionPredicateFilter(filter, regions, filterPersonId);
+
+            var querySearch = from acqFile in Context.PimsAcquisitionFiles.Where(predicate)
+                                .Include(r => r.RegionCodeNavigation)
+                                .Include(p => p.Project)
+                                .Include(s => s.AcquisitionFileStatusTypeCodeNavigation)
+                                .Include(f => f.AcquisitionFundingTypeCodeNavigation)
+                                .Include(ph => ph.AcqPhysFileStatusTypeCodeNavigation)
+                                .Include(t => t.AcquisitionTypeCodeNavigation)
+                                .Include(tm => tm.PimsAcquisitionFilePeople).ThenInclude(c => c.Person)
+                                .Include(ow => ow.PimsAcquisitionOwners)
+                              join fileProp in Context.PimsPropertyAcquisitionFiles
+                                on acqFile.AcquisitionFileId equals fileProp.AcquisitionFileId into filePropGJ
+                              from fp in filePropGJ.DefaultIfEmpty()
+                              join property in Context.PimsProperties
+                                .Include(a => a.Address)
+                                    .ThenInclude(pr => pr.ProvinceState)
+                                .Include(a => a.Address)
+                                    .ThenInclude(pr => pr.Country)
+                                on fp.PropertyId equals property.PropertyId into propGJ
+                              from prop in propGJ.DefaultIfEmpty()
+                              orderby acqFile.AcquisitionFileId, prop.PropertyId
+                              select new AcquisitionFileExportDto()
```

Comment:
I don't know what others think @asanchezr or @FuriousLlama but I can't get behind this syntax. This also seems to be conflating getting data into entities with transforming entities into a display format all within a repository which seems like a violation of separation of concerns.
I would prefer to see:
1) controller handles formatting models, validating requests and returning http responses
2) service handles composing repo calls and responses, business logic, and authorization.
3) repo handles getting data from database or external services in whatever format that db/service returns that data in.

Reply:
Agreed. I would prefer that the Entity -> DTO conversion is done at the Controller level. The repo should always return DB entities to keep separation of concerns. 
Also a minor note on the export DTO name - we usually do not use "DTO" in their names (even if they are DTOs in purpose). A more fitting name would be **AcquisitionFileExportModel**

Reply:
Forgot to mention that I don't think it would be necessary to create a Mapster mapping for this export model conversion. If you look at Devin's PR you can pass the entity in the constructor and create an export model from it. I am using that pattern as well in my export H120 story. 

It would make sense to use mappings for things that are meant to be shared/composed. These "export" models are one-offs so no need to create mappings for them (in my opinion)

Reply:
Should I move the query to the properties repository since the query is returning one row per property in the Acquisition file?